.drone.jsonnet

```diff
@@ -267,16 +267,15 @@ local push_latest = {
 
 local save_artifacts = {
   name: 'save-artifacts',
-  image: 'autonomy/build-container:latest',
+  image: 'mcr.microsoft.com/azure-cli:latest',
```

Comment:
doesn't pur image already has azure-cli?

Reply:
yeah it does. just missed it when I went looking earlier. thx.